src/core/Akka/Actor/Inbox.Actor.cs

```diff
@@ -105,49 +105,57 @@ public bool MessagePredicate(object message)
         /// <returns>TBD</returns>
         protected override bool Receive(object message)
         {
-            switch (message)
-            {
-                case Get get:
+            /*var getMessage = message is Get;
+            var selectMessage = message is Select;
+            var startWatch = message is StartWatch;
+            var stopWatch = message is StopWatch;
+            var kick = message is Kick;*/
+            //var 
+            //Console.WriteLine("Entered the switch");
+            if(message != null){
+                if(message is Get){
                     if (_messages.Count == 0)
                     {
-                        EnqueueQuery(get);
+                        EnqueueQuery((Get)message);
```

Comment:
Please use the new syntax and define a variable name for the casted get directly in the if statement, I.e. „ if ( message is Get get)“